config/routes.rb

```diff
@@ -246,7 +246,8 @@
   post 'contact/dob' => 'contacts#dob_create'
 
   get '/regulations' => 'regulations#show', id: 'index'
-  get '/regulations/wca-regulations-and-guidelines', to: redirect('https://regulations.worldcubeassociation.org/wca-regulations-and-guidelines.pdf', status: 302)
+  get '/regulations/wca-regulations-appended-guidelines', to: redirect('https://regulations.worldcubeassociation.org/wca-regulations-appended-guidelines.pdf', status: 302)
+  get '/regulations/full/wca-regulations-merged-guidelines', to: redirect('https://regulations.worldcubeassociation.org/wca-regulations-merged-guidelines.pdf', status: 302)
```

Comment:
Is the extra '/full' intentional?

Reply:
Yes, this solution requires less changes to the regulations compiler.